src/WCSimRunAction.cc

```diff
@@ -181,34 +181,34 @@ void WCSimRunAction::BeginOfRunAction(const G4Run* /*aRun*/)
   // flat branches, only arrays for PMTs themselves
   // define variables in header, so I can fill them in a separate function.
 
-  geomTree->Branch("GeometryType",geo_type_string,"GeometryType[20]/Char_t");         //example: for std::string data_ : tree->Branch(branchname.c_str(), (void*)data_->c_str(),leafdescription.c_str());
-  geomTree->Branch("CylinderRadius",&cyl_radius,"CylinderRadius/Double_t");
-  geomTree->Branch("CylinderLength",&cyl_length,"CylinderLength/Double_t");
-  geomTree->Branch("PMTtype_ID",pmt_id_string,"PMTtype_ID[50]/Char_t");
-  geomTree->Branch("PMTradius_ID",&pmt_radius_id,"PMTradius_ID/Double_t");
-  geomTree->Branch("PMTtype_OD",pmt_od_string,"PMTtype_OD[50]/Char_t");
-  geomTree->Branch("PMTradius_OD",&pmt_radius_od,"PMTradius_OD/Double_t");
-  geomTree->Branch("numPMT_ID",&numPMT_id,"numPMT_ID/Int_t");
-  geomTree->Branch("numPMT_OD",&numPMT_od,"numPMT_OD/Int_t");
-  geomTree->Branch("Orientation",&orient,"Orientation/Int_t");
-  geomTree->Branch("Offset_x",&offset_x,"Offset_x/Double_t");
-  geomTree->Branch("Offset_y",&offset_y,"Offset_y/Double_t");
-  geomTree->Branch("Offset_z",&offset_z,"Offset_z/Double_t");
+  geomTree->Branch("GeometryType",geo_type_string,"GeometryType[20]/C");         //example: for std::string data_ : tree->Branch(branchname.c_str(), (void*)data_->c_str(),leafdescription.c_str());
```

Comment:
I guess these are what need to be checked with older ROOT versions?

Reply:
resolved in #10